core/templates/dev/head/css/oppia.css

```diff
@@ -346,6 +346,7 @@ body {
 */
 html p, body p {
   margin-top: -5px;
+  display: inline;
```

Comment:
This will affect all p tags in the entire app. Is it possible to do something that's less general, in order not to risk breaking anything?